Directory.Build.props

```diff
@@ -150,6 +171,10 @@
     </Otherwise>
   </Choose>
 
+  <PropertyGroup>
+    <VersionFileForPackages Condition="'$(VersionFileForPackages)' == ''">$(ArtifactsObjDir)version.txt</VersionFileForPackages>
```

Comment:
Given you are adding this did you hook-up something to generate this file?

Reply:
I didn't, I think I brought this in when doing versioning & forgot to check for output. Given that we didn't have a version.txt in the old Standard packages, I'll just remove this.